FirebaseCore/Tests/Unit/FIRHeartbeatLoggerTests.m

```diff
@@ -125,6 +125,30 @@ - (void)testFlushing_UsingV2API_WhenHeartbeatsAreStored_ReturnsNonEmptyPayload {
               }];
 }
 
+- (void)testFlushingAsync_UsingV2API_WhenHeartbeatsAreStored_ReturnsNonEmptyPayload API_AVAILABLE(
```

Comment:
This is actually infeasible since the API belongs to `FirebaseCoreExtension` and core extension is not available in the Firebase Core unit test suite because of how extension is designed to sit on top of core.

Reply:
Another reason to make `FirebaseCoreExtension` a fully fledged module with implementations and its own tests.